environment/src/main/java/com/sequenceiq/environment/environment/service/sdx/SdxPollerService.java

```diff
@@ -72,7 +72,7 @@ private void executeSdxOperationAndStartPolling(Long envId, String environmentNa
             List<String> sdxCrns = getExecuteSdxOperationsAndGetCrns(environmentName, sdxOperation, skipStatuses);
             if (CollectionUtils.isNotEmpty(sdxCrns)) {
                 Polling.stopAfterAttempt(attempt)
-                        .stopIfException(true)
+                        .stopIfException(false)
```

Comment:
Can we have proper exception handling in the related attempt maker and or in the related re-trying client?
This seems to be a bad decision as it would eat/hold pollers until the end of the entire polling period and not to mention about what the rational behind to continue a polling after many 50x response code?

Reply:
I wouldn't like to put exception handling/sorting logic in the attemptmaker. I would rather create a new StopStrategy but I think that is a different issue.
I've talked with @topolyai5 and we agreed that the best solution in this case is polling the flow status instead of the cluster status.

Reply:
Hi @bbihari,
Yes we talked about you will poll the flow id and the poller can be stopped if an exception occurs. Could you please add that change to the PR?

Reply:
done